demo/template.html

```diff
@@ -2,7 +2,7 @@
     <h1>NG2-formly example: Introduction</h1>
     <p>This is a small subset of the things that formly can do :-) See the other examples and the documentation for more.</p>
     <hr>
-    <formly-form [model]="user" [fields]="userFields" (formSubmit)="console($event)" [form]="form">
+    <formly-form [viewModel]="user" [fields]="userFields" (formSubmit)="console($event)" [form]="form">
```

Comment:
This was changed because viewModel is now defined as part of FormlyCommon. If we don't want to change public API we have two options.

1. Update FormlyCommon to use model instead of viewModel. But I feel view Model is correct convention for FormlyField and FormlyGroups as it track state of view. Also I agree model would be right convention for FormlyForm.

2. Use Model for public API but internal track state within viewModel.

I will spend some time on this and will update PR. But do let me know what everybody thinks. I am open for suggestions.